freeipa.spec.in

```diff
@@ -48,10 +48,9 @@
 %global selinux_policy_version 3.14.1-14
 %global slapi_nis_version 0.56.1-4
 %global python_ldap_version 3.1.0-1
-# python3-lib389
-# Fix for "Installation fails: Replica Busy"
-# https://pagure.io/389-ds-base/issue/49818
-%global ds_version 1.4.0.16
+# Fix for create suffix
+# https://pagure.io/389-ds-base/issue/49984
+%global ds_version 1.4.0.21
```

Comment:
This value is set for RHEL but I don't think there is a RHEL build for 1.4.0.21 yet.

Reply:
I removed the RHEL specific changes and left 1.4.0.16 in.